Common/gui/guidefines.h

```diff
@@ -72,7 +72,8 @@ enum GuiVersion
     kGuiVersion_331         = 117,
     kGuiVersion_340         = 118,
     kGuiVersion_350         = 119,
-    kGuiVersion_Current     = kGuiVersion_350,
+    kGuiVersion_399         = 120,
+    kGuiVersion_Current     = kGuiVersion_399,
```

Comment:
Same goes for GuiVersion, as GUI game data format is not modified, only runtime save format.